src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -377,11 +376,13 @@ class boss_svala : public CreatureScript
                             break;
                         case EVENT_FINISH_RITUAL:
                             me->SetDisableGravity(false, true);
-                            SetCombatMovement(true);
-
+                            me->GetMotionMaster()->MoveFall();
+                            me->SetReactState(REACT_AGGRESSIVE);
                             if (Unit* target = me->SelectNearestPlayer(100.0f))
+                            {
                                 AttackStart(target);
-
+                                me->GetMotionMaster()->MoveChase(target);
```

Comment:
Looks like `AttackStart(target);` already handles this

Reply:
nope, need movechase after motionmaster clear

Reply:
you do not kill the motionmaster via clear.  You do me->SetReactState(REACT_PASSIVE) and  me->AttackStop() after that. that will take of blocking chase movement. UpdateVictim will handle the chase movement on its own when aggressive again.

Reply:
thx ;)